src/atoms/Tab/Tab.native.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { View, TouchableOpacity } from 'react-native';
+import styled from 'styled-components/native';
+import Flex from '../Flex';
+import Text from '../Text';
+import { getColor, getIconKeys } from '../../_helpers/theme';
+import Icon from '../Icon';
+import Space from '../Space';
+import automation from '../../_helpers/automation-attributes';
+import isEmpty from '../../_helpers/isEmpty';
+
+const styles = {
+  label: {
+    borderBottomWidth({ isActive }) {
+      if (isActive) {
+        return '1px';
+      }
+      return '0px';
+    },
+    borderBottomColor({ isActive, theme, disabled }) {
+      if (disabled) {
+        if (isActive) {
+          return getColor(theme, 'primary.930');
+        }
+      }
+      if (isActive) {
+        return getColor(theme, 'primary.800');
+      }
+      return 'transparent';
+    },
+    margin() {
+      return [0, 0, 0, 0.5];
+    },
+    padding() {
+      return [1, 0, 1, 0];
+    },
+    color({ isActive, disabled, isPressed }) {
+      if (isPressed) {
+        if (isActive) {
+          return 'primary.900';
+        }
+        return 'shade.970';
+      } else if (isActive) {
+        return disabled ? 'primary.700' : 'primary.800';
+      } else {
+        return disabled ? 'shade.940' : 'shade.960';
+      }
+    },
+  },
+  tab: {
+    backgroundColor({ theme, isActive, isPressed }) {
+      if (isPressed) {
+        if (isActive) {
+          return getColor(theme, 'primary.920');
+        }
+        return getColor(theme, 'tone.940');
+      }
+      return 'transparent';
+    },
+  },
+};
+
+const StyledTabButton = styled(TouchableOpacity)`
+  min-width: 48px;
```

Comment:
```suggestion
  makePxValue(6);
```
Use pixel helper function.